### ISSUE TITLE
Unknown attribute camunda:calledElementVersionTag

### ISSUE THREAD

**Issue by tasso94** [camunda/camunda-modeler#1074]
__Describe the Bug__
The following warning can be seen in the log:
```
Imported BPMN diagram with 1 warning  [warning]
> unknown attribute <camunda:calledElementVersionTag>  [warning]
 ```

__Steps to Reproduce__

1. Create a Call Activity in a BPMN file
2. In Properties Panel...
   1. Select `BPMN` as call activity type
   2. Specify any value for Called Element Input
   3. Select `versionTag` as Binding
   4. Specify any value for Version Tag
3. Save the BPMN file and close it
4. Open the BPMN file again
5. Open the Log

__Expected Behavior__

The warning is not shown.

__Environment__
Camunda Modeler Version: 2.2.3 (2.2.3.2044)

**Comment by nikku** [comment]
Seems like the [extension](https://docs.camunda.org/manual/7.10/reference/bpmn20/custom-extensions/extension-attributes/#calledelementversiontag) is not implemented in [camunda-bpmn-moddle](https://github.com/camunda/camunda-bpmn-moddle).

**Comment by nikku** [comment]
Needs fix via https://github.com/camunda/camunda-bpmn-moddle/issues/43.

**Comment by nikku** [comment]
Closed via https://github.com/camunda/camunda-modeler/commit/1cd510d548b5004ed6116193b4b220759f334f8c.

**Comment by nikku** [comment]
Thanks @pinussilvestrus!

**Comment by nikku** [comment]
Released via `Camunda Modeler v2.2.4`.

**Comment by peterm20192019** [comment]
unknown attribute <label> [ warning ]
